src/fleet_rlm/scaffold/skills/volume-bootstrap/references/filesystem-contract.md

```
@@ -0,0 +1,78 @@
+# Filesystem Contract
+
+## What `ensure_daytona_volume_layout()` Creates
+
+Called automatically during `acreate_workspace_session()` when `volume_name` is set.
+
+Creates (idempotent — safe to call multiple times, never overwrites existing content):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)